### PR TITLE
[Holy Paladin] Moving things around in the checklist

### PR DESCRIPTION
**Pull request by abelito75**
Removed somethings, moved somethings, cat jammed

### CODE REVIEW

**Comment by abelito75** [comment]
this will fail. i fix

File: src/parser/paladin/holy/modules/checklist/Component.js [hunk 1]
```diff
@@ -240,10 +303,11 @@ const HolyPaladinChecklist = ({ combatant, castEfficiency, thresholds, owner })
           </Trans>
         }
       >
-        <Requirement
+        {/*until blizzard figures out mana for a paladin we don't need this*/}
+        {/*<Requirement
           name={<Trans id="paladin.holy.modules.checklist.manaLeft">Mana left</Trans>}
           thresholds={thresholds.manaLeft}
```
Comment (Pewtro) [inline src/parser/paladin/holy/modules/checklist/Component.js]: then delete it

File: src/parser/paladin/holy/CombatLogParser.js [hunk 2]
```diff
@@ -82,8 +86,8 @@ class CombatLogParser extends CoreCombatLogParser {
     hammerOfWrath: HammerOfWrath,
 
     // Generic healer things
-    manaLevelChart: ManaLevelChart,
-    manaUsageChart: ManaUsageChart,
+    // manaLevelChart: ManaLevelChart,
+    // manaUsageChart: ManaUsageChart,
 
     // Features
```
Comment (Pewtro) [inline src/parser/paladin/holy/CombatLogParser.js:86]: delete

Comment (abelito75) [reply src/parser/paladin/holy/CombatLogParser.js:86]: then i just have to re-add it when blizzard finally learns about mana and paladin

Comment (Pewtro) [reply src/parser/paladin/holy/CombatLogParser.js:86]: Correct

**Pull request merged**